driver-scala/src/main/scala/org/mongodb/scala/model/package.scala

```diff
@@ -162,6 +162,12 @@ package object model {
     def apply(): CreateCollectionOptions = new com.mongodb.client.model.CreateCollectionOptions()
   }
 
+  /**
+   * Represents the  commit quorum specifies how many data-bearing members of a replica set, including the primary, must
+   * complete the index builds successfully before the primary marks the indexes as ready.
+   */
+  type CreateIndexCommitQuorum = com.mongodb.CreateIndexCommitQuorum
```

Comment:
Move from `driver-scala/src/main/scala/org/mongodb/scala/model/package.scala` to  `driver-scala/src/main/scala/org/mongodb/scala/package.scala`

Reply:
Done